### PR TITLE
Finishing details

### PR DESCRIPTION
**Pull request by ondrasalek**
Header style edit: created hamburger menu

Footer style edit: centering Kontakty

Body style: hero sizes on screen resize, and margin content

### CODE REVIEW

File: src/components/articles/Article.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import IArticleProps from '../../lib/interfaces/IArticleProps';
 import { Wrapper } from './Article.styled';
+import { isMobile } from 'react-device-detect';
 
 const Article = ({ children, margin }: IArticleProps) => {
     return (
-        <Wrapper primary={margin}>
+        <Wrapper primary={margin} isMobile={isMobile}>
```
Comment (tomesfilip) [inline src/components/articles/Article.tsx:7]: I'd say, there is no need for this parameter, and can be easily adjusted using media queries in Article.styled.tsx

File: src/components/layout/header/Header.styled.tsx [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
+        width: ${size}px;
+        height: auto;
+        border-radius: 50 %;
+`);
+export const Title = styled.h1<{
+    color?: string;
+}>(
+    ({ color }) => css`
+        font-size: 1.5em;
+        font-weight: normal;
```
Comment (tomesfilip) [inline src/components/layout/header/Header.styled.tsx:26]: same here, rather use numeric value instead of normal

**Review by tomesfilip — CHANGES_REQUESTED** [review]
- by any chance, don't we have SU logos in .svg format? would be better..
- idk if we want to have .css files still present or if we want to fully use the potential of styled components. Instead of index.css there might be global.ts file in some /styles folder with global css definitions.

**Comment by ondrasalek** [comment]
Everything @tomesfilip comments is fixed now

File: .github/workflows/firebase-hosting-merge.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# This file was auto-generated by the Firebase CLI
```
Comment (SedlarDavid) [inline .github/workflows/firebase-hosting-merge.yml:1]: Its better to separate context of files into separate PRs 
e.g.
PR for firebase addition
PR for refactoring
PR for styling
and so on, but  its just a hint

**Review by SedlarDavid — APPROVED** [review]

**Review by tomesfilip — APPROVED** [review]
GJ

**Pull request merged**